service-workers/service-worker/unregister-then-register-new-script.https.html

```diff
@@ -72,113 +72,54 @@
       .catch(unreached_rejection(t));
 }, 'Registering a new script URL while an unregistered registration is in use');
```

Comment:
Might as well make this async_test a promise_test too. Usually our files are all promise_test or all async_test so people might assume the promise_tests are isolated from the rest of the file.